mitmproxy/addons/proxyauth.py

```diff
@@ -40,7 +40,7 @@ def load(self, loader):
             "username:pass",
             "any" to accept any user/pass combination,
             "@path" to use an Apache htpasswd file,
-            or "ldap[s]:url_server_ldap[:port]:dn_auth:password:dn_subtree" for LDAP authentication.
+            or "ldap[s]:url_server_ldap:port:dn_auth:password:dn_subtree:ldap_search_filter_key" for LDAP authentication.
```

Comment:
This breaks backward-compatibility for existing users. Can we add a separate `proxyauth_ldap_filter_key` option maybe? I guess alternatively we could add a `?search_filter_key=...` suffix, but not sure if that's any better.

Reply:
Thank you very much for your suggestion. I have recommitted.